Feature/pak 2633

Jonifmi opened the pull request:
Custom images can be added to /img/custom by replacing the old image with new image after:

- copying contents of /img/ to /img/custom 
- setting customIcons: true

Commit:
User can add custom images for warnings

Commit:
removed transparent folder from custom since it can just be copied from img folder

Commit:
update readme

nakkim commented on capmap-config.js:
```diff
@@ -27,6 +27,7 @@ var alertOptions = {
     iconWidth: 30,
     iconHeight: 30,
     transparentIcons: false,
+    customIcons: true,
```
I'd suggest that you should either remove this variable from the "default" configuration file or set it to false, at the moment user gets by broken images by default and it might be confusing.

![Screenshot 2023-08-07 at 8 26 14](https://github.com/fmidev/smartalert-web/assets/12380474/afee15a8-b1ed-4e39-84d1-457db8abcb96)

Commit:
Changed custom icons default setting
custom icon variable is now false by default to prevent user from getting broken images
